TrenDAP/Controllers/TrenDAPDBController.cs

```diff
@@ -95,22 +94,32 @@ public TrenDAPDBController(IConfiguration configuration)
         #endregion
 
         #region [ Http Methods ]
-        [HttpGet, Route("{dataSourceID:int}/{table?}")]
-        public virtual ActionResult GetTable(int dataSourceID, string table = "")
+        [HttpGet, Route("{sourceID:int}/{table?}/{type?}")]
+        public virtual ActionResult GetTable(int sourceID, string table = "", string type = "data")
```

Comment:
I think we are entangling Datasources here. 
There should be a separate Controller for each actual data source implementation if it needs it. 
So a separate `openHistorianController` and a `XDATrend`  Controller etc... all in a separate Folder.

Reply:
Controllers for different implementations seperated